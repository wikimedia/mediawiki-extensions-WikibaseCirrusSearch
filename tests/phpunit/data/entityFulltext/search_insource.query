{
	"search": "insource: \"word1 word2\"",
	"userLang": "en",
	"ns": [0]
}
