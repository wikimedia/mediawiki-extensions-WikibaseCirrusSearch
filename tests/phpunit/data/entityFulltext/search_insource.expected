{
    "query": {
        "bool": {
            "must": [
                {
                    "match_all": {}
                }
            ],
            "filter": [
                {
                    "bool": {
                        "must": [
                            {
                                "query_string": {
                                    "query": "\"word1 word2\"",
                                    "fields": [
                                        "source_text.plain"
                                    ],
                                    "default_operator": "AND",
                                    "allow_leading_wildcard": true,
                                    "fuzzy_prefix_length": 2,
                                    "rewrite": "top_terms_boost_1024"
                                }
                            },
                            {
                                "terms": {
                                    "namespace": [
                                        0
                                    ]
                                }
                            }
                        ]
                    }
                }
            ]
        }
    },
    "rescore_query": [
        {
            "window_size": 8192,
            "query": {
                "query_weight": 1,
                "rescore_query_weight": 1,
                "score_mode": "multiply",
                "rescore_query": {
                    "function_score": {
                        "functions": [
                            {
                                "field_value_factor": {
                                    "field": "incoming_links",
                                    "modifier": "log2p",
                                    "missing": 0
                                }
                            }
                        ]
                    }
                }
            }
        }
    ]
}